fedimint-server/src/net/p2p_connector.rs

```diff
@@ -58,26 +50,55 @@ pub struct TlsConfig {
 }
 
 /// TCP connector with encryption and authentication
-#[derive(Debug)]
 pub struct TlsTcpConnector {
     cfg: TlsConfig,
-    p2p_bind_addr: SocketAddr,
     peers: BTreeMap<PeerId, SafeUrl>,
     identity: PeerId,
+    listener: TcpListener,
+    acceptor: TlsAcceptor,
```

Comment:
We re-use the tls acceptor, as it doesn't change and not need to be reconstructured every time. :+1: